src/Libraries/PythonNodeModels/PythonEngineSelector .cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Linq;
+
+namespace PythonNodeModels
+{
+    /// <summary>
+    /// Enum of possible values of python engine versions.
+    /// TODO: Remove when dynamic loading logic is there then we no longer need a hard copy of the options
+    /// </summary>
+    public enum PythonEngineVersion
+    {
+        IronPython2,
+        CPython3
+    }
+
+    /// <summary>
+    /// Singleton class that other class can access and use for query loaded Python Engine info.
+    /// TODO: Make the Singleton class public when Dynamic loading is also ready.
+    /// </summary>
+    internal sealed class PythonEngineSelector
+    {
+        /// <summary>
+        /// Use Lazy<![CDATA[PythonEngineSelector ]]> to make sure the Singleton class is only initialized once
```

Comment:
Did you forget the `<` and `>`? Also it might be easier to encode the individual characters like `Lazy&lt;PythonEngineSelector&gt;`